packages/dashboard-frontend/src/services/factory-location-adapter/index.ts

```diff
@@ -54,7 +54,7 @@ export class FactoryLocationAdapter implements FactoryLocation {
   }
 
   public static isHttpLocation(href: string): boolean {
-    return /^(https?:\/\/.)[-a-zA-Z0-9@:%._+~#=]{2,}\b([-a-zA-Z0-9@:%_+.~#?&/={},]*)$/.test(href);
+    return /^(https?:\/\/.)[-a-zA-Z0-9@:%._+~#=]{2,}\b([-a-zA-Z0-9@:%_+.~#?&/={}, ]*)$/.test(href);
```

Comment:
@vinokurig could you please add a unit test for that ?

Reply:
done